src/test/java/seedu/address/logic/commands/EligiblePeopleCommandTest.java

```diff
@@ -88,13 +91,13 @@ public void execute_validIndex_noPersonFound() {
     }
 
     @Test
-    public void execute_validIndex_singlePersonFound() {
-        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 1);
+    public void execute_validIndex_eligiblePeopleFound() {
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 4);
         EligiblePeopleCommand command = new EligiblePeopleCommand(INDEX_SECOND_POLICY);
         Policy policy = model.getFilteredPolicyList().get(INDEX_SECOND_POLICY.getZeroBased());
         PersonEligibleForPolicyPredicate predicate = new PersonEligibleForPolicyPredicate(policy);
         expectedModel.updateFilteredPersonList(predicate);
         assertListPeopleCommandSuccess(command, model, expectedMessage, expectedModel);
-        assertEquals(Arrays.asList(ALICE), model.getFilteredPersonList());
+        assertEquals(Arrays.asList(ALICE, CHLOE, VICTORIA, NATASHA), model.getFilteredPersonList());
```

Comment:
@olivercheok20 
hey oli, jus a suggestion - i think u can use readable variable names for the hardcoded values in this test case. reason being tat its more easily maintained by other devs as they wud know wat values to change. Read the best prac on hard coding unit tests [here](https://softwareengineering.stackexchange.com/questions/126652/should-unit-test-expected-results-be-hardcoded)

u can take at my test file for reference. `ModelManagerTest.java`